.ci/install.sh

```diff
@@ -13,8 +13,10 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
         eval "$(pyenv init -)"
     fi
 
-    pyenv install 3.7.1
-    pyenv virtualenv 3.7.1 conan
+    # Google's depot_tools don't work well with new python versions. For now, we
+    # will use -- the soon to be deprecated -- python 2.7 as a workaround.
+    pyenv install 2.7.16
```

Comment:
Our automation will update this at one point in the future automatically again to Python 3.7.1+ (few days to weeks) 🤔

Reply:
Mmh, is there another place that is not in generated scripts where we could force the Python version down to 2.7?